plugins/services/src/js/structs/Framework.js

```diff
@@ -41,6 +41,17 @@ module.exports = class Framework extends Application {
     return tasksSummary;
   }
 
+  getTaskCount() {
+    // TODO: Circular reference workaround DCOS_OSS-783
+    const MesosStateStore = require("#SRC/js/stores/MesosStateStore");
```

Comment:
Let's extract this as a method i.e. `getAllTasks` and reuse it in `getTaskCount` and `getResources`

```javascript
const MesosStateStore = require("#SRC/js/stores/MesosStateStore");
return MesosStateStore.getTasksByService(this) || [];
```